examples/daal/cpp_sycl/source/logistic_regression/logistic_regression_online.hpp

```diff
@@ -0,0 +1,219 @@
+/* file: logistic_regression_online.hpp */
```

Comment:
in all examples we use the extension *.h for headers.

```suggestion
/* file: logistic_regression_online.h */
```